google-ads-annotation-processing/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This is the build configuration for the google-ads-annotation-processing
+ * subproject.
+ */
+
+plugins {
+    id 'com.google.api-ads.java-conventions'
+}
+
+dependencies {
+    implementation 'com.squareup:javapoet:1.11.1'
+    implementation 'javax.annotation:javax.annotation-api:1.3.2'
+    implementation 'com.google.api:gax:1.50.1'
```

Comment:
Does gax need to be here, given it's in java-conventions?

Reply:
I actually added this in only to match the dependencies from the Maven POM files: https://github.com/googleads/google-ads-java/blob/e94a364ccd146706af58a09c0ce35dd5eeace4d2/google-ads-annotation-processing/pom.xml#L66. This one is 1.50.1, while the rest are 1.60.1.

This line can be removed if this subproject doesn't need to be pegged to 1.50.1.

Reply:
Done.